go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -602,13 +664,13 @@ func (be *BuiltinBackupEngine) restoreFile(ctx context.Context, params RestorePa
 	}
 
 	// Check the hash.
-	hash := hasher.HashString()
+	hash := bw.HashString()
 	if hash != fe.Hash {
 		return vterrors.Errorf(vtrpc.Code_INTERNAL, "hash mismatch for %v, got %v expected %v", fe.Name, hash, fe.Hash)
 	}
 
 	// Flush the buffer.
-	if err := dst.Flush(); err != nil {
+	if err := bw.Close(); err != nil {
```

Comment:
nit: fix error message below to match the code change.

Reply:
Sorry to be nitpicky, but I don't think this was addressed? It's line 540 now.

Reply:
It had been addressed when you wrote the comment, I think you may have been looking at the wrong line. And you didn't approve the PR. :sweat: